quadhoral-openmeteo/test_quadhoral_extract.py

```diff
@@ -0,0 +1,128 @@
+'''Contains tests for the extract script in this folder'''
```

Comment:
Good use of test classes, to categorise tests into related groups. Also very readable this way.